instagram/src/PostInfo.js

```diff
@@ -0,0 +1,38 @@
+import React, {Component} from 'react';
+import Comments from './Comments';
+import './App.css';
+import { Panel, Image } from 'react-bootstrap';
+
+export default class PostInfo extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            post: {},
+            comments: []
+        }
+    }
+    componentDidMount() {   
+        this.setState({post: this.props.post}),
```

Comment:
You can combine these two setState functions into one line. 
`this.setState({post: this.props.post, comments: this.props.post.comments});`